maxim/logger/components/generation.py

```
@@ -238,13 +265,20 @@ def add_message_(writer: LogWriter, id: str, message: GenerationRequestMessage):
+        if len(attachments) > 0:
+            for attachment in attachments:
+                Generation.add_attachment_(writer, id, attachment)
         BaseContainer._commit_(
-            writer, Entity.GENERATION, id, "update", {"messages": [message]}
+            writer, Entity.GENERATION, id, "update", {"messages": messages}
         )
 
-    def add_message(self, message: GenerationRequestMessage):
-        self.messages.append(message)
-        self._commit("update", {"messages": [message]})
+    def add_message(self, message: GenerationRequestMessage) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)